File: sdks/python/bolt_prediction_sdk/models/label_import_result_schema.py
"""
LabelImportResultSchema 模型
"""

from typing import Optional, Any, Dict, List
from datetime import datetime
from pydantic import Field

from .base import SDKBaseModel


class LabelImportResultSchema(SDKBaseModel):
    """标注导入结果"""

    total: Optional[int] = Field(description="总行数", default=0)
    imported: Optional[int] = Field(description="成功导入数", default=0)
    skipped: Optional[int] = Field(description="跳过数", default=0)
    duplicates: Optional[int] = Field(description="重复数", default=0)
    errors: Optional[int] = Field(description="错误数", default=0)
    error_details: Optional[Any] = Field(description="错误详情", default=None)

File: sdks/java/src/main/java/com/boltprediction/sdk/model/LabelImportResultSchema.java
package com.boltprediction.sdk.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import java.util.List;
import java.util.Map;
import java.time.OffsetDateTime;

/** 标注导入结果 */
@JsonIgnoreProperties(ignoreUnknown = true)
public class LabelImportResultSchema {

    @JsonProperty("total")
    private Integer total;

    @JsonProperty("imported")
    private Integer imported;

    @JsonProperty("skipped")
    private Integer skipped;

    @JsonProperty("duplicates")
    private Integer duplicates;

    @JsonProperty("errors")
    private Integer errors;

    @JsonProperty("error_details")
    private Object errorDetails;

    public Integer getTotal() {
        return total;
    }

    public void setTotal(Integer total) {
        this.total = total;
    }

    public Integer getImported() {
        return imported;
    }

    public void setImported(Integer imported) {
        this.imported = imported;
    }

    public Integer getSkipped() {
        return skipped;
    }

    public void setSkipped(Integer skipped) {
        this.skipped = skipped;
    }

    public Integer getDuplicates() {
        return duplicates;
    }

    public void setDuplicates(Integer duplicates) {
        this.duplicates = duplicates;
    }

    public Integer getErrors() {
        return errors;
    }

    public void setErrors(Integer errors) {
        this.errors = errors;
    }

    public Object getErrorDetails() {
        return errorDetails;
    }

    public void setErrorDetails(Object errorDetails) {
        this.errorDetails = errorDetails;
    }

}